.github/workflows/publish_image.yml

```diff
@@ -140,3 +140,58 @@ jobs:
     with:
       version: ${{ needs.settings.outputs.image_version }}
     secrets: inherit
+
+  update-remote-repo:
+    needs:
+      - settings
+      - publish
+    runs-on: ubuntu-latest
+    env:
+      service_to_update: 'studio'
+      cli_repo_owner: supabase
+      cli_repo: cli
+      cli_repo_main_branch: 'main'
+      cli_repo_pr_title: 'New Studio Version'
+      github_user: 'Ivan Vasilov'
+      github_user_email: 'vasilov.ivan@gmail.com'
+
+    steps:
+      - name: Setup Git config
```

Comment:
I think this is a lot of implicit information and I'd rather have the CLI repo handle the majority of this, in case they change something.

What I'd prefer, is triggering a custom event and then have a github action in the CLI repo that reacts to this event.

Example step to dispatch an event


```

 - name: Dispatch
        uses: peter-evans/repository-dispatch@v2.1.0
        with:
          # This action creates repository_dispatch events. The default GITHUB_TOKEN token can only be used if you are dispatching
          # the same repository that the workflow is executing in. To dispatch to a remote repository you must create a Personal
          # Access Token (PAT) with the repo scope and store it as a secret. If you will be dispatching to a public repository
          # then you can use the more limited public_repo scope.
          token: ${{ secrets.NPM_TOKEN_XYZ }}
          repository: ${{ matrix.repository }}
          event-type: studio-released
          client-payload: '{"version": "${{ steps.last_release.outputs.tag_name }}", "message": "${{ github.event.head_commit.message }}"}'
```

Reply:
The CLI could then react to it with a new action

```
  repository_dispatch:
    types: [studio-released]

```

Ideally, CLI can then use their regular package manager